### PR TITLE
Labyrinth Anna Hasselström & Nathalie Nordström

### PR DESCRIPTION
**Pull request by nathalienordstrom**
https://labyrint-game.netlify.app

### CODE REVIEW

**Comment by nathalienordstrom** [comment]
a

**Review by Snabbina — CHANGES_REQUESTED** [review]
🤩 Hey, Anna and Nathalie!

What a mysterious game you've made!

You have worked well with the layout in the various media queries, and nice elements in the styling where there is a consistent theme in both color and shapes.

The main features are included which allows the game to go through from start to finish. and also the opportunity to go back a step or to start over is a practical feature.
Great job with the styled components! The naming of functions and variables is clear and consistent through out the code.

Also very impressed with your solution with the conditional rendering for the different components. 

We have a changes requested before approval though:
- Not consistent with ;, semikolon. For example in App.js and PlayingGame.js you use ; but in GameOver and loading you are not using them anymore. Change to one approach.
- indentation is missing in some places. 

Please see our comments, questions and extra appreciation for your code!
🎉

Best of the best from Sabina and Ellen!

**Comment by nathalienordstrom** [comment]
All done thank you Sabina and Ellen

File: code/src/components/StartingGame.js
Comment (HIPPIEKICK) [reply code/src/components/StartingGame.js]: I agree, the start button should be hidden or disabled if it's not possible to start the game 👍

File: code/src/components/GameOver.js
Comment (HIPPIEKICK) [reply code/src/components/GameOver.js:19]: Yes, this should be indented. And no need for the fragment

File: code/src/components/Loading.js
Comment (HIPPIEKICK) [inline code/src/components/Loading.js:8]: Great name of a boolean!

File: code/src/components/PlayingGame.js
Comment (HIPPIEKICK) [reply code/src/components/PlayingGame.js:99]: Goes for this whole file

File: code/src/reducers/currentstate.js
Comment (HIPPIEKICK) [inline code/src/reducers/currentstate.js:36]: Maybe `goBack` would be a fitting name?

File: code/src/reducers/currentstate.js
Comment (HIPPIEKICK) [inline code/src/reducers/currentstate.js:2]: ;

File: code/src/reducers/ui.js
Comment (HIPPIEKICK) [inline code/src/reducers/ui.js:1]: Semicolons missing in this file ;)

**Comment by HIPPIEKICK** [comment]
Hi Anna and Nathalie! Please check indentation and semicolons in all files. Also, I would like to ask you to think about your starting screen. Right now it's not clear that you have to click Save before Play Game. Once you've saved - you get no indicator of that the save was successful. One suggestion is to only have the Save button visible to start with, and then after the save is done, that section is hidden and the start screen only shows the Play Game button. Alternatively, you could display a message to the user saying that the save was successful and apply some more styling to the disabled Play Game button so the user knows it's not possible to start.

**Pull request closed without merging**
